### PR TITLE
README: add contributing guidelines

### PR DESCRIPTION
**Pull request by ricardosalveti**

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -2,3 +2,16 @@ Meta-LMP
 ================================
 
 Collection of layers used by the Linux microPlatform project.
+
+Contributing
+------------
+
+Please submit any patches against the `meta-lmp-base` and `meta-lmp-bsp`
+layers by using the GitHub pull-request feature. Fork the repo, make a branch,
+do the work, rebase from upstream and create the pull request.
```
Comment (kprosise) [inline README.md:11]:
```suggestion
do the work, then rebase from upstream and create the pull request.
```

File: README.md [hunk 2]
```diff
@@ -2,3 +2,16 @@ Meta-LMP
...
+
+For some useful guidelines to be followed when submitting patches,
+please refer to:
```
Comment (kprosise) [inline README.md:14]:
```suggestion
For some useful guidelines when submitting patches, please refer to:
```

**Review by kprosise — APPROVED** [review]
Just a couple of suggestions, but it looks good!

**Review by quaresmajose — APPROVED** [review]

**Pull request merged**